install.sh

```diff
@@ -35,7 +35,7 @@ FLAGS:
         --force-local-kube   Force reinstall local Kubernetes cluster if it is already installed
         --force-kubectl      Force reinstall kubectl client if it is already installed
         --force-kind         Force reinstall Kind if it is already installed
-        --docker-mirror      Use docker mirror to pull image, dockerhub.azk8s.cn => docker.io, gcr.azk8s.cn => gcr.io
+        --docker-mirror      Use docker mirror to pull image, dockerhub.azk8s.cn => docker.io, gcr.azk8s.cn => gcr.io; ghcr.io are NOT supported now.
```

Comment:
BTW, I heard these mirrors are only available only for Azure China network. (But they have nothing to do with this PR.)

Reply:
Here is a related isue: https://github.com/chaos-mesh/chaos-mesh/issues/2294

Reply:
It seems the policy of azk8s.cn has been more strict: I can not even pull `dockerhub.azk8s.cn/library/nginx`, server responses 403